src/azure-cli/azure/cli/command_modules/vm/tests/latest/recordings/test_disk_encryption_set_update.yaml

```diff
@@ -131,7 +131,7 @@ interactions:
       User-Agent:
       - AZURECLI/2.21.0 azsdk-python-azure-mgmt-keyvault/8.0.0 Python/3.7.4 (Windows-10-10.0.19041-SP0)
     method: PUT
-    uri: https://management.azure.com/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/cli_test_disk_encryption_set_update_000001/providers/Microsoft.KeyVault/vaults/vault1-000002?api-version=2019-09-01
+    uri: https://management.azure.com/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/cli_test_disk_encryption_set_update_000001/providers/Microsoft.KeyVault/vaults/vault1-000002?api-version=2021-04-01-preview
```

Comment:
Please make sure it can pass, or mark it if it fails. I do not recommend editing recording file. Just run it.

Reply:
sure, i will rerun live test for all the test related